### PR TITLE
Rename 'context' to 'loginfo' in the logger module

### PR DESCRIPTION
**Pull request by miaoyq**
/cc @thaJeztah @justincormack 

Signed-off-by: Yanqiang Miao <miao.yanqiang@zte.com.cn>

### CODE REVIEW

**Comment by vdemeester** [comment]
@miaoyq `gofmt` error 👼 

```
05:30:17 Successfully built f049b54837ce
05:30:19 Congratulations!  All commits are properly signed with the DCO!
05:30:19 These files are not properly gofmt'd:
05:30:19  - daemon/logger/gelf/gelf.go
05:30:19 
05:30:19 Please reformat the above files using "gofmt -s -w" and commit the result.
```

**Comment by miaoyq** [comment]
Thanks @vdemeester 
Fixed

**Comment by justincormack** [comment]
Another `lint` error
```
13:19:30 Errors from golint:
13:19:30 daemon/logger/loginfo.go:10:1: comment on exported type LogInfo should be of the form "LogInfo ..." (with optional leading article)
13:19:30 
13:19:30 Please fix the above errors. You can test via "golint" and commit the result.
13:19:30 
```

File: api/types/container/config.go [hunk 1]
```diff
@@ -34,29 +34,29 @@ type HealthConfig struct {
 // All fields added to this struct must be marked `omitempty` to keep getting
 // predictable hashes from the old `v1Compatibility` configuration.
 type Config struct {
-	Hostname        string                // Hostname
```
Comment (justincormack) [inline api/types/container/config.go]: confused as to why spacing changed here, does not seem to be changed in this PR.

Comment (miaoyq) [inline api/types/container/config.go]: The lint error have been fixed, Thanks so much! @justincormack

**Comment by miaoyq** [comment]
@vdemeester @justincormack  @AkihiroSuda Could this PR be merged?

File: api/types/container/config.go (hunk 1 above)
Comment (vdemeester) [inline api/types/container/config.go]: @miaoyq as it's not related to the changes, could you remove this file from the commit ? 👼 (should be `git checkout HEAD~1 api/types/container/config.go`)

File: daemon/logger/loginfo.go [hunk 2]
```diff
@@ -7,8 +7,8 @@ import (
 	"time"
 )
 
-// Context provides enough information for a logging driver to do its function.
-type Context struct {
+// LogInfo provides enough information for a logging driver to do its function.
```
Comment (vdemeester) [inline daemon/logger/loginfo.go]: As we already are in `logger` package, `LogInfo` seems redondant, maybe just `Info` ? @thaJeztah wdyt 👼 ?

**Review by vdemeester — CHANGES_REQUESTED** [review]

File: daemon/logger/loginfo.go (hunk 2 above)
Comment (thaJeztah) [inline daemon/logger/loginfo.go]: Sounds good to me

Comment (miaoyq) [inline daemon/logger/loginfo.go]: OK, I will change `LogInfo` to `Info`, Thanks @vdemeester @thaJeztah

Comment (miaoyq) [inline daemon/logger/loginfo.go]: @vdemeester @thaJeztah
updated. PTAL

Comment (miaoyq) [inline daemon/logger/loginfo.go]: @vdemeester @thaJeztah 
Updated.  PTAL

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**